src/Symfony/Component/HttpClient/composer.json

```diff
@@ -25,7 +25,7 @@
         "php": ">=8.1",
         "psr/log": "^1|^2|^3",
         "symfony/deprecation-contracts": "^2.5|^3",
-        "symfony/http-client-contracts": "~3.4.4|^3.5.2",
+        "symfony/http-client-contracts": "~3.4.5|^3.5.3",
```

Comment:
this looks unneeded to me as neither 3.4.4 nor 3.5.2 have been released yet

Reply:
Is this different from the other change I made to http-client-contracts a few days ago?
https://github.com/symfony/symfony/pull/59062#discussion_r1865776359